### PR TITLE
Update metadata for tile_xyzservices.py

### PR DESCRIPTION
**Pull request by Ajoke23**
Update metadata

### CODE REVIEW

File: examples/topics/geo/tile_xyzservices.py [hunk 1]
```diff
@@ -1,3 +1,16 @@
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map. The 'tile_xyz' service provides map tiles, 
+which are typically used to display maps visualizations and geographical location in web applications.The data used to create the plot is 
+sourced from public XYZ tile services.This example demonstrates how to create a plot in Bokeh with a map background using tile_providers.
+The chart shows a map of a specific geographical region, which is defined by the range bounds supplied in web mercator coordinates.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.Figure, bokeh.tile_providers.get_provider, bokeh.tile_provider.Vendors
+    :more info: [Geographical Plot](https://docs.bokeh.org/en/latest/docs/user_guide/topics/geo.html)
```
Comment (bryevdv) [inline examples/topics/geo/tile_xyzservices.py]: This needs to be a [Sphinx](https://www.sphinx-doc.org/en/master/) reference, so something like:
```suggestion
    :refs: :ref:`ug_topics_geo_tile_provider_maps`
```
These references can be viewed in the ReST source for the docs, e.g. for this one: https://raw.githubusercontent.com/bokeh/bokeh/branch-3.2/docs/bokeh/source/docs/user_guide/topics/geo.rst

File: examples/topics/geo/tile_xyzservices.py [hunk 2]
```diff
@@ -1,3 +1,16 @@
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map. The 'tile_xyz' service provides map tiles,
```
Comment (bryevdv) [inline examples/topics/geo/tile_xyzservices.py]: This needs a little adjustment. The package that provides the tiles services is [`xyzservivces`](https://github.com/geopandas/xyzservices) (and not 'tile_xyz') we might want to link to the `xyzservices` project here. Also it's not a hard and fast rule, but I've been trying to keep the line length of docstring text for examples closer to 80-100 characters wide, so I'd suggest shortening the lines somewhat.

File: examples/topics/geo/tile_xyzservices.py [hunk 3]
```diff
@@ -1,3 +1,16 @@
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map. The 'tile_xyz' service provides map tiles, 
+which are typically used to display maps visualizations and geographical location in web applications.The data used to create the plot is 
+sourced from public XYZ tile services.This example demonstrates how to create a plot in Bokeh with a map background using tile_providers.
```
Comment (bryevdv) [inline examples/topics/geo/tile_xyzservices.py]: missing space
```suggestion
sourced from public XYZ tile services. This example demonstrates how to create a plot in Bokeh with a map background using tile_providers.
```

File: examples/topics/geo/tile_xyzservices.py [hunk 4]
```diff
@@ -1,3 +1,16 @@
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map. The 'tile_xyz' service provides map tiles, 
+which are typically used to display maps visualizations and geographical location in web applications.The data used to create the plot is
```
Comment (bryevdv) [inline examples/topics/geo/tile_xyzservices.py]: missing space
```suggestion
which are typically used to display maps visualizations and geographical location in web applications. The data used to create the plot is 
```

**Review by bryevdv — CHANGES_REQUESTED** [review]

**Comment by bryevdv** [comment]
Thanks for the PR @Ajoke23 just a few comments above cc @tcmetzger who may also want to review

File: examples/topics/geo/tile_xyzservices.py (hunk 1 above)
Comment (Ajoke23) [reply examples/topics/geo/tile_xyzservices.py]: Thanks for the feedback. I have made changes regarding this

File: examples/topics/geo/tile_xyzservices.py (hunk 4 above)
Comment (Ajoke23) [reply examples/topics/geo/tile_xyzservices.py]: Changes made @bryevdv

File: examples/topics/geo/tile_xyzservices.py (hunk 2 above)
Comment (Ajoke23) [reply examples/topics/geo/tile_xyzservices.py]: Thanks a lot for the feedback. @bryevdv.
I noticed the code regarding the plot is showing error so can i just create a PR or i need to create it as an issue first before PR

File: examples/topics/geo/tile_xyzservices.py (hunk 3 above)
Comment (Ajoke23) [reply examples/topics/geo/tile_xyzservices.py]: Changes made

**Comment by Ajoke23** [comment]
@bryevdv kindly check this. I added metadata to legend_hide
https://github.com/bokeh/bokeh/pull/12986/commits/548d587b692d12ca4b5e5d07c4dde6dad0b8e84c

File: examples/topics/geo/tile_xyzservices.py [hunk 5]
```diff
@@ -1,10 +1,28 @@
-import xyzservices.providers as xyz
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map.
+The xyzservices provides map tiles,which are typically used to display maps visualizations
```
Comment (tcmetzger) [inline examples/topics/geo/tile_xyzservices.py]:
```suggestion
The xyzservices library provides map tiles, which are typically used to display maps visualizations 
```

File: examples/topics/geo/tile_xyzservices.py [hunk 6]
```diff
@@ -1,10 +1,28 @@
-import xyzservices.providers as xyz
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map.
+The xyzservices provides map tiles,which are typically used to display maps visualizations 
+and geographical location in web applications.The data used to create the plot is
```
Comment (tcmetzger) [inline examples/topics/geo/tile_xyzservices.py]:
```suggestion
and geographical location in web applications. The data used to create the plot is 
```

File: examples/topics/geo/tile_xyzservices.py [hunk 7]
```diff
@@ -1,10 +1,28 @@
-import xyzservices.providers as xyz
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map.
+The xyzservices provides map tiles,which are typically used to display maps visualizations 
+and geographical location in web applications.The data used to create the plot is 
+sourced from public XYZ tile services.This example demonstrates how to create a plot in Bokeh
```
Comment (tcmetzger) [inline examples/topics/geo/tile_xyzservices.py]:
```suggestion
sourced from public XYZ tile services. This example demonstrates how to create a plot in Bokeh
```

**Review by tcmetzger — CHANGES_REQUESTED** [review]
Thank you, just some small change requests in the text!

File: examples/topics/geo/tile_xyzservices.py (hunk 7 above)
Comment (Ajoke23) [reply examples/topics/geo/tile_xyzservices.py]: @tcmetzger All changes have been made. Kindly review

**Comment by bryevdv** [comment]
@Ajoke23 This PR seems to be duplicated of #12992 which one should we consider and keep open?

**Comment by Ajoke23** [comment]
> @Ajoke23 This PR seems to be duplicated of #12992 which one should we consider and keep open?

Pls consider #12992

**Comment by bryevdv** [comment]
superseded by #12992

**Pull request closed without merging**